### PR TITLE
week1 homework done

### PR DESCRIPTION
**Pull request by hmeljana**
uspelaa! :D

### CODE REVIEW

File: createOrganization.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+
```
Comment (bdimitrovski) [inline createOrganization.json:2]: Why is this file empty?

File: navigation.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+"header" : {
+    "goToHomeScreenLogo" : ".vs-c-site-logo",
+    "showSearchButton" : ".vs-c-project-search .vs-c-btn",
+    "hideSearchButton" : ".vs-is-active .vs-c-btn",
+    "searchInput" : ".vs-c-project-search__input input[type='text']",
+
```
Comment (bdimitrovski) [inline navigation.json:7]: Blank space

File: navigation.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "showNotificationsButton" : ".el-badge .vs-c-btn",
+    "showAllNotifications" : "span[class='vs-c-notifications-show-all vs-u-cursor--pointer']",
+    "notificationSettings" : ".vs-u-display--flex span[title='Notification Settings']",
+
```
Comment (bdimitrovski) [inline navigation.json:11]: Blank space

File: navigation.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    "howItWorksButton" : ".vs-l-project__options > .el-tooltip.vs-c-btn.vs-c-btn--link"
+},
+
+
```
Comment (bdimitrovski) [inline navigation.json:15]: Blank space

File: navigation.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    "emailInput" : "input[type='email']",
+    "passwordInput" : "input[type='password']",
+    "submitButton" : ".vs-u-text--left button[type='submit']",
+    "errorMessage" : ".vs-c-form-item__error-wrapper .el-form-item-error--top",
+    "loginViaGoogle" : ".vs-c-auth-modal__body-social-wrapper .vs-c-btn--gp",
+    "loginViaFB" : ".vs-c-auth-modal__body-social-wrapper .vs-c-btn--fb",
+    "loginViaTwitter" : ".vs-c-auth-modal__body-social-wrapper .vs-c-btn--tw",
+    "loginViaRegzen" : ".vs-c-auth-modal__body-social-wrapper .vs-c-btn--regzen"
+},
+
```
Comment (bdimitrovski) [inline navigation.json:27]: Same as above

File: sidebar.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    "myOrganization" : {
+        "myOrganizationLink" : ".vs-c-list__item .vs-c-list__organisation",
+        "expandMyOrganization" : "button[slot='toggle-button']",
+        "myBoard" : ".vs-c-list--boards li[data-type='board']",
+        "starBoard" : ".vs-c-list__item-board .vs-c-list__btn .vs-c-icon",
+        "addBoardIconHover" : ".vs-c-list-btn--add-new",
+        "addNewBoard" : "div:nth-of-type(14) .vs-c-list a"
+
+    } 
+}
```
Comment (bdimitrovski) [inline sidebar.json:21]: Newline

File: register.json [hunk 7]
```diff
@@ -0,0 +1,9 @@
+{
+    "backToPricing" : ".vs-u-gap--bottom a[href='https://cypress-api.vivifyscrum-stage.com/pricing']",
+    "showHidePassword" : ".el-input button[type='button']",
+    "numberOfUsers" : ".el-input input[placeholder='Number of people in your team (51-100)']",
+    "checkbox" : "span[class='vs-c-checkbox-check']",
+    "goToLoginLink" : "a[href='/login']",
+    "pricingPlanMonthlyOption" : ".vs-c-switch-pricing-plan-option p"
+
+}
```
Comment (bdimitrovski) [inline register.json:9]: Newline

File: createBoard.json [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+    "selectOrg" : ".el-input input",
+    "dropdownOrgOption" : "li.vs-c-select-dropdown__item",
+    "page3" : ".vs-c-dot-pagination > li:nth-of-type(3)",
+    "page4" : ".vs-c-dot-pagination > li:nth-of-type(4)",
+    "page5" : ".vs-c-dot-pagination > li:nth-of-type(5)",
+    "boardTypeScrum" : ".vs-c-radio span[name='type_scrum']",
+    "boardTypeKanban" : ".vs-c-radio span[name='type_kanban']",
+    "importConfigFrom" : "div:nth-of-type(1) > .vs-input-border > .el-select.vs-c-new-board-select > .el-input > .el-input__inner",
+    "configFromDetailsHover" : ".vs-c-new-board-tooltip",
+    "imortTeamMembersFrom" : "div:nth-of-type(2) > .vs-input-border > .el-select.vs-c-new-board-select > .el-input > .el-input__inner"
```
Comment (bdimitrovski) [inline createBoard.json:11]: importTeamMembersFrom

File: createBoard.json [hunk 9]
```diff
@@ -0,0 +1,13 @@
+{
+    "selectOrg" : ".el-input input",
+    "dropdownOrgOption" : "li.vs-c-select-dropdown__item",
+    "page3" : ".vs-c-dot-pagination > li:nth-of-type(3)",
```
Comment (bdimitrovski) [inline createBoard.json:4]: What are these?

File: navigation.json [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+    "nameInput" : ".el-form-item input[type='text']",
+    "backButton" : "button[name='prev_btn']",
+    "nextButton" : "button[name='next_btn']",
+    "page1" : ".vs-c-dot-pagination > li:nth-of-type(1)",
+    "page2" : ".vs-c-dot-pagination > li:nth-of-type(2)",
+    "uploadLogo" : ".el-upload-dragger",
+    "closeModalButton" : "button.vs-c-close-modal"
+}
+
+}
```
Comment (bdimitrovski) [inline navigation.json:39]: Newline

File: navigation.json (hunk 2 above)
Comment (hmeljana) [reply navigation.json:7]: @bdimitrovski Does the "blank space" refer to empty space between classes? If so, these are mostly classes from different elements (parent el class + child el class). ChroPath doesn’t recognize elements if there's no space between classes.

File: createBoard.json (hunk 8 above)
Comment (bdimitrovski) [reply createBoard.json:11]: There's a typo in `imortTeamMembersFrom`, it should be `importTeamMembersFrom`

File: register.json (hunk 7 above)
Comment (hmeljana) [reply register.json:9]: @bdimitrovski Can you please be more specific?

File: navigation.json (hunk 2 above)
Comment (bdimitrovski) [reply navigation.json:7]: Yes, blank space between `searchInput` and the next element. Nothing major, but still would be better not to have blank lines unnecessarily if possible.

File: createOrganization.json (hunk 1 above)
Comment (bdimitrovski) [reply createOrganization.json:2]: Then please delete this file if it's not used anymore.

File: createBoard.json (hunk 9 above)
Comment (hmeljana) [reply createBoard.json:4]: @bdimitrovski 
The "page3"  (to page5) is for pagination, so we can check which one is active during the creation process.
The "dropdownOrgOption" is for one Organization I have created from the dropdown menu.
The "selectOrg" is the input field that should be used for entering the Organization name OR for expanding the dropdown menu.

**Pull request merged**